response.go

```diff
@@ -177,6 +143,49 @@ func (r *Response) Attachment(dataType AttachmentType, url string) error {
 	return err
 }
 
+// AttachmentData sends an image, sound, video or a regular file to a chat via an io.Reader.
+func (r *Response) AttachmentData(
```

Comment:
Please make this function signature all one line, so it follows the conventions of the rest of the code base.